app/src/main/AndroidManifest.xml

```diff
@@ -9,8 +9,10 @@
         android:label="@string/app_name"
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
+        android:requestLegacyExternalStorage="true"
```

Comment:
Since this is required only for android 10.
Can we add this conditionally or add it with a comment?